tools/node.js

```diff
@@ -73,11 +73,13 @@ foam.CLASS({
     function execute() {
       var modelArgs = this.modelArgs;
       var X = this.__subContext__;
-      return X.arequire(this.modelId).then(function(model) {
-        return model.create(modelArgs, X).execute();
-      }).catch(function(err) {
-        console.log(err.stack);
-      });
+      var modelId = this.modelId;
+      return X.arequire(modelId)
+        .catch(console.log)
```

Comment:
They changed this in Chrome a couple releases ago, but this will break in browsers other than Chrome. Actually, I see that it works in Firefox also.